Week 8: Portfolio - Alma

almaherris opened the pull request:
## Netlify link
https://almaherrstrom-portfolio.netlify.app/

## Collaborators
[almaherris]

Commit:
Scaffold project

Commit:
Create components

Commit:
Add tickertape/banner

Commit:
Started filling components with content

Commit:
Add content to contact section

Commit:
rearrange components

Commit:
add colors

Commit:
work on adding correct fonts, colors, font size

Commit:
added wavy line at projects

Commit:
put profile pic in resuables

Commit:
added arror in intro section

Commit:
installed react-icons and added new arrow

Commit:
added links in the contact section

Commit:
changed some root colors

Commit:
styled photo components

Commit:
added images to public folder

Commit:
added scroll fucntion to arrow in intro section

Commit:
added and styled tags

Commit:
more tag styling

Commit:
added buttons

Commit:
styled buttons

Commit:
set up css grid for project cars

Commit:
project cards done

Commit:
article card done

Commit:
updated article button styling

Commit:
added hovering affect to buttons and links

Commit:
added transition effect to hovering

Commit:
tablet responsiveness

Commit:
responsiveness desktop and accessibility updates

Commit:
split up css in components

Commit:
added node_modules to gitignore

Commit:
Remove node_modules from tracking

Commit:
Update contact icons for tablet

Commit:
added mockup images of projects

Commit:
updated colors

Commit:
fixed accessibility

Commit:
added a project and updated some images

Commit:
fixed type in link

Commit:
updated text in projects

Commit:
updated wording

Martin-Joensson commented on src/App.css:
```diff
@@ -0,0 +1,260 @@
+  --color-7: #ffde30;
+  --contact-icon-grey: #d0d0d0;
+  --link-btn-grey: #f1eeee;
+}
```
I really like that you made variables for the colors! They might benefit from more descriptive names of where they are used.

Martin-Joensson commented on src/App.css:
```diff
@@ -0,0 +1,260 @@
+    grid-template-columns: repeat(4, 1fr);
+    gap: 10px;
+  }
+}
```
I really like your App.css. It's very easy to follow, and smart use of multiple classnames that need the same styling.

Martin-Joensson commented on src/components/sections/Banner.jsx:
```diff
@@ -0,0 +1,14 @@
+      </p>
+    </div>
+  )
+}
```
If you make a second <p> with the same classname and the exact same content you should be able to get the infinitely looping effect. The second <p> will be hidden because of your overflow: hidden but will follow the text you have now. Since it will be the exact same the user won't notice when the animation restarts and everything "jumps back" to the start.

I hope that made sense... :P

almaherris replied:
Tried this but it didn't work 😕

Martin-Joensson commented on src/components/sections/ProjectCard.jsx:
```diff
@@ -0,0 +1,47 @@
+        }
+      )}
+    </div>
+  )
```
Very nice reusable component!

Martin-Joensson commented on src/components/sections/Intro.css:
```diff
@@ -0,0 +1,41 @@
+  
+  .intro p {
+    font-size: 18px;
+  }
```
On a very large screen your Intro text becomes really wide, it might be good to put a width: 75ch or some other way of keeping it more condensed and easier to read.

Martin-Joensson reviewed (commented):
I really like you portfolio! The subtle gradient on the Tech and skills backgrounds are a nice touch! Chefs kiss!

/Martin

HIPPIEKICK reviewed (changes requested):
⭐️
- Really nice looking animations!
- I personally really like your color choices, especially the firey one 🍅

**Needs another take**
- Make sure to follow the distances (margins, paddings) of the design. One example: It should be 32px between the image and the ”Hi, I’m Alma Herrström Frontend developer” text and same distance to the paragraph below. And same between paragraph text and arrow. Double check all distances please.
- Make sure to account for devices between 320px and 1600px. Your project images are too wide for 320px phones.
- Tablet design: The Tech section should be a bit more narrow, the Featured Projects section should be "image left, text right” layout, Skills section should be two columns, SoMe icons should be closer together
- Small desktop design: Looks good apart from what’s mentioned before with margin/padding. Make sure all sections are aligned. Also, the border for images should be thicker.

Commit:
added new project

Commit:
updates for tablet

Commit:
more margin updates and removed redux from list

Commit:
intro layout

Commit:
add project

Commit:
small desktop margin/padding updates

Commit:
large desktop updates

Commit:
margins padding and intro section on desktop

Commit:
remove comments file

Commit:
some more details on padding etc

Commit:
padding arrow

almaherris commented:
I think I have done the necessary updates, let me know if it's still not ok 🤓 Updated following:
✅It should be 32px between the image and the ”Hi, I’m Alma Herrström Frontend developer” text and same distance to the paragraph below. And same between paragraph text and arrow. - updated
✅Double check all distances please. - updated

✅ Make sure to account for devices between 320px and 1600px. Your project images are too wide for 320px phones. - fixed responsiveness for 320px phones

Tablet design:
✅ Tech section should be a bit more narrow - made it a bit more narrow
✅ Featured Projects section should be "image left, text right” layout - it already has this layout, I chose the same breakpoint as in the design (744 px)
✅ Skills section should be two columns - updated desktop breakpoint so ipad pro has the 2 column layout
✅ SoMe icons should be closer together - put them closer

Small desktop design:
✅ Looks good apart from what’s mentioned before with margin/padding. - updated margins/paddings as well as I could from the design, it's hard to see at some places for example bottom of the arrow
✅ Make sure all sections are aligned. - not entirely sure what this means but they are aligned with design, some sections are aligned to left and some are centered, I think I got it right
✅ Also, the border for images should be thicker - made borders thicker

HIPPIEKICK commented:
Make sure the sections are aligned, e.g. in mobile the sections should have 16px on the sides. And in desktop, they seem to have different padding to the sides too. Have a look at it when you have time, but I will approve now - this is well improved so good job!

almaherris committed:
added article
